dlt/destinations/impl/databricks/databricks.py

```diff
@@ -0,0 +1,336 @@
+        "bigint": "BIGINT",
+        "binary": "BINARY",
+        "decimal": "DECIMAL",  # DECIMAL(p,s) format
+        "time": "STRING",
+    }
+
+    dbt_to_sct = {
+        "STRING": "text",
+        "DOUBLE": "double",
+        "BOOLEAN": "bool",
+        "DATE": "date",
+        "TIMESTAMP": "timestamp",
+        "BIGINT": "bigint",
+        "INT": "bigint",
+        "SMALLINT": "bigint",
+        "TINYINT": "bigint",
+        "BINARY": "binary",
+        "DECIMAL": "decimal",
+    }
+
+    sct_to_dbt = {
+        "decimal": "DECIMAL(%i,%i)",
+        "wei": "DECIMAL(%i,%i)",
+    }
+
+    def to_db_integer_type(
+        self, precision: Optional[int], table_format: TTableFormat = None
+    ) -> str:
+        if precision is None:
+            return "BIGINT"
+        if precision <= 8:
+            return "TINYINT"
+        if precision <= 16:
+            return "SMALLINT"
+        if precision <= 32:
+            return "INT"
+        return "BIGINT"
+
+    def from_db_type(
+        self, db_type: str, precision: Optional[int] = None, scale: Optional[int] = None
+    ) -> TColumnType:
+        # precision and scale arguments here are meaningless as they're not included separately in information schema
+        # We use full_data_type from databricks which is either in form "typename" or "typename(precision, scale)"
+        type_parts = db_type.split("(")
+        if len(type_parts) > 1:
+            db_type = type_parts[0]
+            scale_str = type_parts[1].strip(")")
+            precision, scale = [int(val) for val in scale_str.split(",")]
+        else:
+            scale = precision = None
+        db_type = db_type.upper()
+        if db_type == "DECIMAL":
+            if (precision, scale) == self.wei_precision():
+                return dict(data_type="wei", precision=precision, scale=scale)
+        return super().from_db_type(db_type, precision, scale)
+
+
+class DatabricksLoadJob(LoadJob, FollowupJob):
+    def __init__(
+        self,
+        file_path: str,
+        table_name: str,
+        load_id: str,
+        client: DatabricksSqlClient,
+        stage_name: Optional[str] = None,
+        staging_credentials: Optional[CredentialsConfiguration] = None,
+    ) -> None:
+        file_name = FileStorage.get_file_name_from_file_path(file_path)
+        super().__init__(file_name)
+
+        qualified_table_name = client.make_qualified_table_name(table_name)
+
+        # extract and prepare some vars
+        bucket_path = (
+            NewReferenceJob.resolve_reference(file_path)
+            if NewReferenceJob.is_reference_job(file_path)
+            else ""
+        )
+        file_name = (
+            FileStorage.get_file_name_from_file_path(bucket_path) if bucket_path else file_name
+        )
+        from_clause = ""
+        credentials_clause = ""
+        files_clause = ""
+        # stage_file_path = ""
+        format_options = ""
+        copy_options = "COPY_OPTIONS ('mergeSchema'='true')"
```

Comment:
is mergeschema needed? normally the loader should have updated the destination schema before parquet files start getting copied into there.